Skills/Perk/FertilitySubmissiveAndBreedable.gd

```diff
@@ -0,0 +1,22 @@
+extends PerkBase
+
+func _init():
+	id = Perk.FertilitySubmissiveAndBreedable
+	skillGroup = Skill.Fertility
+
+func getVisibleName():
+	return "Submissive and breedable"
+
+func getVisibleDescription():
+	return "Others view you as a subservient, perfect toy to please their carnal desires, most noticeably breeding. " \
+			+ "Dominant partners may go an extra round or two, and gain a breeder fetish if not having one already."
```

Comment:
remove the fetish gaining. I have others plans for how to implement that, just doing it off of a perk is too easy and unrealistic

Reply:
I asked a week ago if Submissive and Breedable perk is okay, but you didn't reply. https://github.com/Alexofp/BDCC/pull/14#issuecomment-1567950657

Reply:
I was busy/tired and forgot, sorry